sdk/tables/Azure.Data.Tables/src/Extensions/TableClientExtensions.cs

```diff
@@ -16,6 +16,6 @@ public static class TableClientExtensions
         /// <param name="client">The <see cref="TableClient"/>.</param>
         /// <param name="filter">A filter expresssion.</param>
         /// <returns>The string representation of the filter expression.</returns>
-        public static string CreateFilter<T>(this TableClient client, Expression<Func<T, bool>> filter) => client.Bind(filter);
+        public static string CreateFilter<T>(this TableClient client, Expression<Func<T, bool>> filter) => TableClient.Bind(filter);
```

Comment:
Turn this into a static method?

Reply:
Hmm, now I don't recall why it was ever an extension method :)